tests/5.1/allocate/test_calloc_host.c

```diff
@@ -0,0 +1,74 @@
+//===------------------------ test_calloc_host.c --------------------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+//
+// Tests the omp_calloc routine to make sure memory is properly allocated
+// on the host and is zero initialized. Allocates an array and makes sure
+// each value is set to 0 to start. Then adds i to each index and validates the
+// changes persisted and were written correctly.
+//
+//===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include "ompvv.h"
+
+#define N 1024
+
+int test_omp_calloc_host() {
+  int errors = 0;
+  int *x;
+
+  omp_memspace_handle_t  memspace = omp_default_mem_space;
+  omp_alloctrait_t       traits[0] = {};
+  omp_allocator_handle_t alloc = omp_init_allocator(memspace,0,traits);
+
+  x = (int *)omp_calloc(64, N*sizeof(int), alloc);
+
```

Comment:
I think you should add something like:
`if (x == NULL) {
        OMPVV_ERROR("omp_calloc returned null");
        return (1);
  }
`

Reply:
Use predefined memory allocator omp_default_mem_alloc here, especially since the traits are not being modified.